.travis/build.sh

```diff
@@ -10,6 +10,7 @@ export COMPAT=${COMPAT:-1}
 export PATH=$CWD/dependencies/bin:"$HOME"/.local/bin:"$PATH"
 export TIMEOUT=180
 export PYTEST_PAR=2
+export PYTHONPATH=$PWD/contrib/pylightning:$PYTHONPATH
```

Comment:
This has always been working, what is the error you are getting without this?

Reply:
Dynamically-started plugins don't get the non-exported environment variable passed at the command line